heudiconv/utils.py

```diff
@@ -21,12 +21,10 @@
 
 
 seqinfo_fields = [
-    'total_files_till_now',  # 0
+    'series_files',  # 0
     'example_dcm_file',      # 1
     'series_id',             # 2
     'dcm_dir_name',          # 3
-    'unspecified2',          # 4
-    'unspecified3',          # 5
```

Comment:
wouldn't this break any hope for backward compatibility?

Reply:
yes, there are quite a few phased-out designs we could part with.

though we can add some config/argument to re-enable legacy seqinfo fields
some ideas for the next release can be found at https://github.com/nipy/heudiconv/projects/2

Reply:
I am just afraid that many custom heuristics which access fields via positional indexes would get unnecessarily (i.e. we could avoid it) broken.  I would just keep those afloat for now